com.github.Bleuzen.FFaudioConverter.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id": "com.github.Bleuzen.FFaudioConverter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.11",
+    "sdk": "org.kde.Sdk",
+    "rename-icon": "ffaudioconverter",
+    "rename-desktop-file": "ffaudioconverter.desktop",
+    "rename-appdata-file": "FFaudioConverter.appdata.xml",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--filesystem=host"
+    ],
+    "modules": [
+        {
+            "name": "libmp3lame",
+            "config-opts": ["--enable-shared",
+                            "--enable-nasm",
+                            "--bindir=/app/bin"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/lame/lame-3.100.tar.gz",
+                    "sha256": "ddfe36cab873794038ae2c1210557ad34857a4b6bdc515785d1da9e175b1da1e"
+                }
+            ]
+        },
+        {
+            "name": "ffmpeg",
+            "config-opts": [
+                "--enable-gpl",
+                "--disable-static",
+                "--enable-shared",
+                "--disable-doc",
+                "--disable-ffplay",
+                "--disable-ffprobe",
+                "--enable-libmp3lame",
+                "--enable-libvorbis",
+                "--enable-libopus"
+            ],
+            "cleanup": [
+                "/share/ffmpeg/examples"
+            ],
+            "sources": [
+                {
+                "type": "archive",
+                "url": "https://ffmpeg.org/releases/ffmpeg-4.1.tar.xz",
+                "sha256": "a38ec4d026efb58506a99ad5cd23d5a9793b4bf415f2c4c2e9c1bb444acd1994"
+                }
+            ]
+        },
+        {
+            "name": "FFaudioConverter",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "file",
```

Comment:
Should be `archive`.

Reply:
I tried this first, but problem was that in the archive there are some files and a "src" directory. Always when I used archive, I only had access to the src directory. But to build I also need the files outside src. So I changed it to file and extracted it with a command.

Reply:
Add `"strip-components": 0`. But I would say you are generating your tarball in a very strange way that will confuse all distros that try to package this.

Reply:
I added `"strip-components": 0` for now.
Ok, maybe I will change the tar format in the future..